src/tools/pub/publishing.md

```diff
@@ -41,6 +41,13 @@ are a few additional requirements for uploading a package:
   and Git dependencies don't support version resolution as well as hosted
   dependencies do.
 
+* You must have a [Google Account,][Google Account]
+  which pub uses to manage package upload permissions.
```

Comment:
I'm not sure we need to say this at all, just below we have a paragraph saying:
> Be aware that the email address associated with your Google account is
displayed on the [Pub site]({{site.pub}}) along with any packages you upload.

Reply:
It is, however, a prerequisite, so it makes sense to list it here.

Reply:
Fair point :)